BankAPILibrary/GiniBankAPILibraryPinningExample/GiniBankAPILibraryPinningExampleTests/TransferSummaryIntegrationTest.swift

```diff
@@ -0,0 +1,135 @@
+
+                                let fixtureExtractionsJson = self.loadFile(withName: "result_Gini_invoice_example", ofType: "json")
+
+                                let fixtureExtractionsContainer = try! JSONDecoder().decode(ExtractionsContainer.self, from: fixtureExtractionsJson)
+
+                                // Verify we received the correct extractions for this test
+                                XCTAssertEqual(fixtureExtractionsContainer.extractions.first(where: {$0.name == "iban"})?.value, extractions.first(where: {$0.name == "iban"})?.value)
+                                XCTAssertEqual(fixtureExtractionsContainer.extractions.first(where: {$0.name == "paymentRecipient"})?.value, extractions.first(where: {$0.name == "paymentRecipient"})?.value)
+                                XCTAssertEqual(fixtureExtractionsContainer.extractions.first(where: {$0.name == "paymentPurpose"})?.value, extractions.first(where: {$0.name == "paymentPurpose"})?.value)
+                                XCTAssertEqual(fixtureExtractionsContainer.extractions.first(where: {$0.name == "bic"})?.value, extractions.first(where: {$0.name == "bic"})?.value)
+                                XCTAssertEqual(fixtureExtractionsContainer.extractions.first(where: {$0.name == "amountToPay"})?.value, extractions.first(where: {$0.name == "amountToPay"})?.value)
+
+                                // 3. Assuming the user saw the following extractions:
+                                //    amountToPay, iban, bic, paymentPurpose and paymentRecipient
+                                //    Supposing the user changed the amountToPay from "995.00:EUR" to "950.00:EUR"
+                                //    we need to update that extraction
+                                extractions.first(where: {$0.name == "amountToPay"})?.value = "950.00:EUR"
+
+                                //    Send transfer summary for the extractions the user saw
+                                //    with the final (user confirmed or updated) extraction values
+                                self.documentService.submitFeedback(for: compositeDocument, with: extractions) { result in
+                                    switch result {
+                                    case .success(_):
+
+                                        // 4. Verify that the extractions were updated
+                                        self.documentService.extractions(for: compositeDocument, cancellationToken: CancellationToken()) { result in
+                                            switch result {
+                                            case .success(let extractionResult):
+                                                let extractionsAfterFeedback = extractionResult.extractions
+                                                let fixtureExtractionsAfterFeedbackJson = self.loadFile(withName: "result_Gini_invoice_example_after_feedback", ofType: "json")
```

Comment:
[clarification]: do we want to rename the file accordingly from `result_Gini_invoice_example_after_feedback` to `result_Gini_invoice_example_after_transfer_summary`?
Also the properties: `extractionsAfterFeedback`, `fixtureExtractionsAfterFeedbackJson`, `fixtureExtractionsAfterFeedbackContainer`.

Reply:
We keep the file names